.github/workflows/validate.yml

```diff
@@ -0,0 +1,43 @@
+name: Validate
+
+on: [push, workflow_dispatch]
```

Comment:
Is this the typical CI file? If so, shouldn't this have pull_request on it?

Reply:
This one started with the workflow for repolinter in this repo, which isn't triggered by PRs. I'll add it.